test/inference.jl

```diff
@@ -855,4 +862,4 @@ let f, m
         Expr(:return, Expr(:call, Core._apply, :+, SSAValue(0)))
     ]
     @test @inferred(f()) == 6
-end
+end
```

Comment:
Not sure what you mean. Is the fix needed due to an issue with my conflict resolution?